packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -553,6 +566,33 @@ class SemanticsProperties extends DiagnosticableTree {
   ///  * [SemanticsFlag.namesRoute] for a description of how the name is used.
   final bool namesRoute;
 
+  /// If non-null, whether the node represents an image.
+  ///
+  /// See also:
+  ///
+  ///   * [SemanticsFlag.image], for the flag this setting controls.
+  final bool image;
+
+  /// If non-null, whether the node should be considered a live region.
+  ///
+  /// On Android, when a live region semantics node is first created TalkBack
```

Comment:
Can you document what iOS does when this is set?

Reply:
Done